tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt

```diff
@@ -910,4 +910,6 @@ special ;	sc	;	moderate	;	Sardinian
 special ;	sr_Latn	;	modern	;	Serbian (Latn)
 special ;	yrl	;	moderate	;	Nheengatu
 special ;	yue_Hans	;	modern	;	Cantonese (Simplified)
-special  ; uz_Cyrl ; basic ; Uzbek (Cyrillic)
+special ; uz_Cyrl ; basic ; Uzbek (Cyrillic)
+special	;	to	;	modern
```

Comment:
Can we add the language name for this one?

Reply:
Will do. These are Tongo and Quechua